web/docs/introduction/migrate-from-11.md

```diff
@@ -488,6 +804,14 @@ export async function migrateUsernameAuth(prismaClient: PrismaClient) {
 
 ##### Email
 
+:::caution Users will need to reset their password
+
+There is a breaking change between the old and the new auth in the way the password is hashed. This means that users will need to reset their password after the migration, as the old password will no longer work.
+
+It would be best to notify your users about this change and put a notice on your login page to **request a password reset**.
```

Comment:
Can they do this on wrong password? So if users input wrong password, they show them the message? ANy way to do that easily?